Bug: https://bugs.gentoo.org/848525

From aa646fa0ca3e164b09949c546796ec50433b748d Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Tue, 17 May 2022 16:02:49 +0200
Subject: [PATCH] meson: Allow unittest inspector to fail

It is only used to make the test output nicer, and it is completely fine
to fail.

Closes: #187
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index b49d5f0..2e1edd1 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -124,7 +124,7 @@ if os_backend == 'linux' and gobject_introspection.found()
 
     python3 = find_program('python3')
     unittest_inspector = find_program('linux/unittest_inspector.py')
-    r = run_command(unittest_inspector, files('linux/integration-test.py'), check: true)
+    r = run_command(unittest_inspector, files('linux/integration-test.py'), check: false)
     unit_tests = r.stdout().strip().split('\n')
 
     foreach ut: unit_tests
-- 
2.35.1

